subsys/mgmt/ec_host_cmd/ec_host_cmd_handler.c

```diff
@@ -161,10 +161,20 @@ int ec_host_cmd_send_response(enum ec_host_cmd_status status,
 	struct ec_host_cmd_tx_buf *tx = &hc->tx;
 
 	if (status != EC_HOST_CMD_SUCCESS) {
+		const struct ec_host_cmd_request_header *const rx_header =
+			(const struct ec_host_cmd_request_header *const)hc->rx_ctx.buf;
+
+		LOG_INF("HC 0x%04x err %d", rx_header->cmd_id, status);
 		send_status_response(hc->backend, tx, status);
 		return status;
 	}
 
+#ifdef CONFIG_EC_HOST_CMD_LOG_DBG_BUFFERS
```

Comment:
Non-blocking - can you use `if (IS_ENABLED(CONFG...))` here?

Reply:
Yes, done.